survey/README.md

```diff
@@ -0,0 +1,35 @@
+# The Package Manager Security Landscape Survey
+
+In 2022, the OpenSSF Securing Software Repositories Working Group embarked on
+an effort to survey the current security landscape of package manager
+ecosystems.
+
+This effort provides a survey/landscape of different security mechanisms and
+features that are implemented across the different ecosystems as it pertains to
+security critical user journeys. The goal of this exercise is to understand
+overall security priorities, and to cross-pollinate ideas, threat models and
+designs across the ecosystems. 
+
+The survey covers 3 main broad topics: Secure, Ingest and
+Evaluate. This includes areas of:
+- Integrity
+- Typos & dependency confusion
+- Handling compromise and malice
+- Portecting the ecosystem
+- Auth and credential Management
```

Comment:
Nit: should "credential" be "Credential"?

Reply:
made everything lower case for consistency